### PR TITLE
Add multi-threaded test

### PR DESCRIPTION
**Pull request by ldorau**
This change is [Reviewable](https://reviewable.io/reviews/pmem/vmemcache/23)

### CODE REVIEW

**Comment by ldorau** [comment]
This PR contains PR https://github.com/pmem/vmemcache/pull/8

**Comment by janekmi** [comment]
Please rebase.

**Comment by ldorau** [comment]
Rebased

File: tests/vmemcache_test_mt.c [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+		ctx[i].buffs = buffs;
+		ctx[i].nbuffs = nbuffs;
+		ctx[i].ops_count = ops_count / n_threads;
+		ctx[i].thread_number = i;
+	}
+
+	/* run all tests */
+	run_test_put(n_threads, threads, ctx);
+	run_test_get(cache, n_threads, threads, ctx);
+
```
Comment (pbalcer) [inline tests/vmemcache_test_mt.c]: it would be good to have also a mix workload (get/put) and mix with evict workload.

Comment (ldorau) [reply tests/vmemcache_test_mt.c]: It will be added in the next PR, because it requires this fix: #27 to be merged first.

**Comment by ldorau** [comment]
@pbalcer please resolve your 1 discussion

**Pull request merged**